### PR TITLE
Added ActiveCollab version 2.0.0

### PR DESCRIPTION
**Pull request by vives**

### CODE REVIEW

File: src/main/resources/activecollab-config/init.xml [hunk 1]
```diff
@@ -17,30 +17,110 @@
...
-    <parameter name="format"
-               description="The format that the reponse is expected to be."/>
+    <parameter name="apiUrl" description="The url to access the ActiveCollab API."/>
+    <parameter name="apiVersion" description="The version of the ActiveCollab API."/>
+    <parameter name="intent" description="The intent for the token."/>
+    <parameter name="clientVendor" description="The name of the instance."/>
+    <parameter name="clientName" description="The name of the client."/>
+    <parameter name="email" description="The Email ID of the user."/>
+    <parameter name="password" description="The password of an ActiveCollab account."/>
+    <parameter name="email" description="The Email ID of the user."/>
```
Comment (keerthu) [inline src/main/resources/activecollab-config/init.xml:27]: Duplicate parameters. Remove it.

Comment (Niranjan-K) [reply src/main/resources/activecollab-config/init.xml:27]: @hariss63 : Has the duplicate entry been removed?

**Comment by Niranjan-K** [comment]
@hariss63 : A suggestion when you commit is to have different commit title. If not, then it is better to do a git squash and have a single commit.

File: src/main/resources/activecollab-config/init.xml (hunk 1 above)
Comment (vives) [reply src/main/resources/activecollab-config/init.xml:27]: Fixed in https://github.com/wso2-extensions/esb-connector-activecollab/pull/5

**Pull request merged**